core/templates/components/forms/schema-based-editors/schema-based-float-editor.directive.ts

```diff
@@ -122,6 +128,13 @@ angular.module('oppia').directive('schemaBasedFloatEditor', [
             if (ctrl.localValue === undefined) {
               ctrl.localValue = 0.0;
             }
+            // Check customization arg valur of numeric input interaction.
```

Comment:
What is "arg valur"?

Reply:
Please prefer complete words in comments.

Reply:
done